recipe/meta.yaml

```diff
@@ -24,6 +24,7 @@ requirements:
     - pip
     - python
     - cython
+    - setuptools_scm
```

Comment:
That was in the pyproject.toml, right? @marcelotrevisani we really need to tackle that on grayskull :-/

Reply:
Yep, it is in pyproject.toml